modules/javafx.graphics/src/main/java/com/sun/javafx/font/coretext/CTGlyphLayout.java

```diff
@@ -80,6 +80,22 @@ private int getFontSlot(long runRef, CompositeFontResource fr, String name, int
         if (!fontName.equalsIgnoreCase(name)) {
             if (fr == null) return -1;
             slot = fr.getSlotForFont(fontName);
+            if (slot == -1) {
```

Comment:
minor: may be `< 0`?

Reply:
We should never have -2 etc. If we do, then the code will fail and we'd know about it

Reply:
I noticed checks on either `slot >= 0` or `slot == -1` and was thinking about it as well. But I agree with @prrace , there is not an assignment that could legally result in -2. If that happens, it would be a major failure and we can't pretend it is the same as -1.